include/openenclave/corelibc/assert.h

```diff
@@ -4,11 +4,30 @@
 #ifndef _OE_ASSERT_H
 #define _OE_ASSERT_H
 
-#if defined(OE_NEED_STDC_NAMES)
+/*
+**==============================================================================
+**
+** OE names:
+**
+**==============================================================================
+*/
 
+/* Include the oe_assert() definition from enclave.h. */
 #include <openenclave/enclave.h>
-#define assert(EXPR) oe_assert(EXPR)
-#define __assert_fail(EXPR) __oe_assert_fail(EXPR)
+
+/*
+**==============================================================================
+**
+** Standard-C names:
+**
+**==============================================================================
+*/
+
+#if defined(OE_NEED_STDC_NAMES)
+
+#define assert oe_assert
+
+#define __assert_fail __oe_assert_fail
```

Comment:
Curious why this change is needed.

Reply:
These definitions were missing. Corelibc must provide both oe-prefixed names and standard-C names for everything. This change will allow programs such as mbedtls and OpenSSL to compile when calling assert().

Reply:
Great. Thanks for the insight.